source/fuzz/fuzzer_util.h

```diff
@@ -152,10 +152,20 @@ bool IsMergeOrContinue(opt::IRContext* ir_context, uint32_t block_id);
 uint32_t FindFunctionType(opt::IRContext* ir_context,
                           const std::vector<uint32_t>& type_ids);
 
+// Returns a type instruction (OpTypeFunction) for |function|.
+// Returns |nullptr| if type is not found.
+opt::Instruction* GetFunctionType(opt::IRContext* context,
+                                  const opt::Function* function);
+
 // Returns the function with result id |function_id|, or |nullptr| if no such
 // function exists.
 opt::Function* FindFunction(opt::IRContext* ir_context, uint32_t function_id);
 
+// Returns |true| if one of entry points has function id |function_id|
+// TODO: TransformationAddFunctionCall also has this functionality as a static
+// method
```

Comment:
Please open a GitHub issue regarding the fact that TransformationAddFunctionCall also has this functionality (mentioning in that issue that it is being added again by this PR, which you can reference from the issue), and change TODO: to TODO(full url of GitHub issue) - look for other occurrences of TODO in spirv-fuzz and you'll see they do this; we try to avoid a plain old TODO: wherever possible as we just lose track of those.

Reply:
Nit: end function comment with period.

Reply:
Fixed in 2a20e6e

Reply:
Added a new issue #3218.